perf/latency.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Mangos Authors
+// Copyright 2018, 2019 The Mangos Authors
```

Comment:
In mangos, we just use a single year, so change 2018 to 2019.

Reply:
fixed